doc/articles/platform-specific-csharp.md

```diff
@@ -12,12 +12,29 @@ This guide covers multiple approaches to managing per-platform code in C#. See [
 
 ## Project structure
 
-There are two ways to restrict code or XAML markup to be used only on a specific platform:
+There are multiple ways to restrict code or XAML markup to be used only on a specific platform:
 
+* Use conditional code based on the `OperatingSystem.IsXXX` method
 * Use [conditionals](https://learn.microsoft.com/dotnet/csharp/language-reference/preprocessor-directives/preprocessor-if) in a source file
 * Place the code in a separate file which is only included in the desired platform.
 
-The structure of an Uno app created with the default Visual Studio template is [explained in more detail here](uno-app-solution-structure.md).
+The structure of an Uno Platform app created with the default Visual Studio template is [explained in more detail here](xref:Uno.Development.AppStructure).
+
+### OperatingSystem.IsXXX
+
+The Uno Platform templates differentiate platforms using the current target framework, yet it's possible to determine the running platform at runtime using:
+
+```csharp
+if (OperatingSystem.IsBrowser())
+{
+   // Do something WebAssembly specific
+}
+```
+
+When building an application which uses the Skia renderer, it is generally best to use such conditionals in order to build for only one target framework in class libraries (e.g. when only targeting `net9.0`, without a platform specifier like `net9.0-ios`).
+
+> [!NOTE]
+> [JSImport/JSExport](xref:Uno.Wasm.Bootstrap.JSInterop) and [BrowserHtmlElement](xref:Uno.Interop.WasmJavaScript1) are available on all platforms targeting .NET 7 and later, and code usingthose APIs to be conditionally excluded at compile time, and should only use `OperatingSystem` conditions.
 
 ## `#if` conditionals
```

Comment:
@jeromelaban I imagine we are keeping Mac Catalyst for now based on previous versions that may still use it right?
![image](https://github.com/user-attachments/assets/3c33ee00-5281-403d-8197-8c28264409c0)

Reply:
Correct.